config-freebsd.md

```diff
@@ -0,0 +1,124 @@
+* **`ip6`** _(string, OPTIONAL)_ - control the availability of IPv6 addresses.
+    This is typically left unset if the container has a vnet, set to "inherit" to allow access to host (or parent container) addresses or set to "disable" to stop use of IPv6 entirely.
+* **`vnet`** _(string, OPTIONAL)_ - control the vnet used for this container.
+    The value can be "new" which causes a new vnet to be created for the container or "inherit" which shares the vnet for the parent container (or host if there is no parent).
+* **`sysvmsg`** _(string, OPTIONAL)_ - allow access to SYSV IPC message primitives.
+    If set to "inherit", all IPC objects on the system are visible to this container, whether they were created by the container itself, the base system, or other containers.  If set to "new", the container will have its own key namespace, and can only see the objects that it has created; the system (or parent container) has access to the container's objects, but not to its keys.  If set to "disable", the container cannot perform any sysvmsg-related system calls.
+* **`sysvsem`** _(string, OPTIONAL)_ - allow access to SYSV IPC semaphore primitives, in the same manner as sysvmsg.
+* **`sysvshm`** _(string, OPTIONAL)_ - allow access to SYSV IPC shared memory primitives, in the same manner as sysvmsg.
+* **`enforceStatfs`** _(integer, OPTIONAL)_ - control visibility of mounts in the container.
+    A value of 0 allows visibility of all host mounts, 1 allows visibility of mounts nested under the container's root and 2 only allows the container root to be visible. If unset, the default value is 2.
+* **`allow`** _(object, OPTIONAL)_ - Some restrictions of the container environment may be set on a per-container basis.  With the exception of **`setHostname`** and **`reservedPorts`**, these boolean parameters are off by default.
+  - **`setHostname`** _(bool, OPTIONAL)_ - Allow the container's hostname to be changed.
+  - **`rawSockets`** _(bool, OPTIONAL)_ - Allow the container to use raw sockets to support network utilities such as ping and traceroute.
+  - **`chflags`** _(bool, OPTIONAL)_ - Allow the system file flags to be changed.
+  - **`mount`** _(array of strings, OPTIONAL)_ - Allow the listed filesystem types to be mounted and unmounted in the container.
+  - **`quotas`** _(bool, OPTIONAL)_ - Allow the filesystem quotas to be changed in the container.
+  - **`socketAf`** _(bool, OPTIONAL)_ - Allow socket types other than IPv4, IPv6 and unix.
+  - **`reservedPorts`** _(bool, OPTIONAL)_ - Allow the jail to bind to ports lower than 1024.
+  - **`suser`** _(bool, OPTIONAL)_ - The value of the jail's security.bsd.suser_enabled sysctl. The super-user will be disabled automatically if its parent system has it disabled.  The super-user is enabled by default.
+
+### Mapping from jail(8) config file
+
+This table shows how to map settings from a typical `jail(8)` config file to the container configuration:
```

Comment:
Is this advisory/suggestion, or prescriptive? (It currently reads as the former)

Reply:
Good point, I think we can say `This tables defines the mappings from a typical` here. Does that read better?